x-pack/plugins/painless_lab/server/routes/api/execute.ts

```diff
@@ -12,7 +12,16 @@ import { isEsError } from '../../lib';
 const bodySchema = schema.object({
   script: schema.object({
     source: schema.string(),
+    params: schema.maybe(schema.recordOf(schema.string(), schema.any())),
   }),
+  context: schema.maybe(schema.string()),
+  context_setup: schema.maybe(
+    schema.object({
+      params: schema.maybe(schema.any()),
+      document: schema.recordOf(schema.string(), schema.any()),
+      index: schema.string(),
```

Comment:
I made this required, since it looks like ES requires it, but I also opened https://github.com/elastic/kibana/issues/60020 as a follow-up.